### ISSUE TITLE
Code for NPM 1.2.0 version does not match taggeed version in github

### ISSUE THREAD

**Issue by austinnichols101** [nutgaard/react-router-breadcrumbs#226]
It appears that v1.2.0 from npm has bundled code from another version (I think 1.1).  For example, the npm version specifies `"react-router": "2.7.0"` as a dependency.

`npm install react-router-breadcrumbs@1.2.0`

```
**cat node_modules/react-router-breadcrumbs/package.json | grep 1.2.0**
        "raw": "react-router-breadcrumbs@1.2.0",
        "rawSpec": "1.2.0",
        "spec": "1.2.0",
  "_from": "react-router-breadcrumbs@1.2.0",
  "_id": "react-router-breadcrumbs@1.2.0",
    "tmp": "tmp/react-router-breadcrumbs-1.2.0.tgz_1474404409315_0.8684582761488855"
    "raw": "react-router-breadcrumbs@1.2.0",
    "rawSpec": "1.2.0",
    "spec": "1.2.0",
  "_resolved": "https://registry.npmjs.org/react-router-breadcrumbs/-/react-router-breadcrumbs-1.2.0.tgz",
  "_spec": "react-router-breadcrumbs@1.2.0",
    "tarball": "https://registry.npmjs.org/react-router-breadcrumbs/-/react-router-breadcrumbs-1.2.0.tgz"
  "version": "1.2.0"
```

```
**cat node_modules/react-router-breadcrumbs/package.json | grep '"react-router"'**
    "react-router": "2.7.0",
    "react-router",
    "react-router": "^2.0.1"
```

**Comment by nutgaard** [comment]
Hi, and thanks for posting the issue.

The difference between 1.1 and 1.2 are were a simple bugfix, hence the similarities in `package.json`. I've inspected the zip-files, and from what I can see the changes are included in 1.2.

The library does not bundle `react-router` as a dependency at all, but it is listed as a `devDependency` and `peerDependency`.

There has however been some changes since 1.2 that are not released yet, most prominent is allowing `react-router@^3.0.0` as a peerDependency. I'll try to get that one out as soon as possible. 

I will be closing this issue for now, but if I've misunderstood anything please let me know so I can correct any potential bugs. :)

**Comment by austinnichols101** [comment]
My mistake - I was mis-reading devDependencies for dependencies.  However, I'm showing a peer dependency of ^2.0.1 here which is keeping me from running react-router 3.0.0.

https://github.com/nutgaard/react-router-breadcrumbs/commit/b70b6a1c7f4598651f140494526b6ae03a1319ba

**Comment by nutgaard** [comment]
Yeah, I discovered that when going through the commit-history yesterday. Seems like a slip-up on my behalf, and will try to get a new version out today. 

Anyways, I'm reopening this issue to address the issue.

**Comment by nutgaard** [comment]
`react-router-breadcrumbs@1.3.0` released which allows `react-router@^3.0.0` as peerDependency. Closing this issue
